portal-api/src/v1/gef/routes.js

```diff
@@ -1,31 +1,50 @@
 const express = require('express');
 const { validate } = require('../../role-validator');
-const mandatoryCriteriaVersioned = require('./controllers/mandatoryCriteriaVersioned.controller');
+
 const application = require('./controllers/application.controller');
+const exporter = require('./controllers/exporter.controller');
+const facilities = require('./controllers/facilities.controller');
+const mandatoryCriteriaVersioned = require('./controllers/mandatoryCriteriaVersioned.controller');
 
 const router = express.Router();
 
 // Application
 router.route('/application')
-  .get(validate({ role: ['maker', 'checker'] }), application.getAll)
-  .post(validate({ role: ['maker'] }), application.create);
+  .get(validate({ role: ['checker', 'data-admin'] }), application.getAll)
```

Comment:
I've added a comment on Rich's PR about this too. Should a maker also be able to list applications?

Reply:
For me to work on this ticket, I had to merge Matts work into mine, this is why you are seeing PORT API changes. Once Matt has successfully merged his changes into MASTER, this will soon go.

Reply:
I think we'll need to keep them separate in future if possible so you're both working off master. Matts PR has a couple of changes needed and there's danger they'll be overwritten if this PR is merged after Matts. Once Matt has merged, can you rebase your branch before merging into master?

Reply:
Yeah thats fair enough. Merging matts work into mine is not the best solution, and there is a risk of overwriting changes etc. Im just worried that all our tasks will be delayed, as mocking data can only take you so far. If there is a better solution please let me know.

Reply:
It's worth brining up with Gavin. I guess it's the danger of splitting the same function between frontend & backend. Maybe we could have a gef branch that you both branch from and then submit gef branch for PR once you're both ready? 
Given the limited timeframe we could probably live with what we have now and rely on e2e & api tests to highlight merge issues

Reply:
Yeah that could be a good idea. Let me speak to Matt